common/app/common/pages/PageHeader.java

```diff
@@ -0,0 +1,31 @@
+package common.pages;
+
+import common.cms.CmsPage;
+
+public class PageHeader {
```

Comment:
Yes, but not only for handlebars, but for any template. We were already using this same structure for Scala templates, except that now the complexity has increased due to the CMS and Template system.